Gemfile

```diff
@@ -91,8 +91,7 @@ end
 
 group :google, :manageiq_default do
   manageiq_plugin "manageiq-providers-google"
-  gem "fog-google",                     ">=0.5.2",       :require => false
-  gem "google-api-client",              "~>0.8.6",       :require => false
+  gem "fog-google",                     ">=0.5.4",       :require => false
```

Comment:
Do we even need this in manageiq proper anymore?

Reply:
I don't see it referenced anywhere, I'll just remove it then